copis/core.py

```diff
@@ -495,15 +515,18 @@ def remove_action(self, index: int) -> Action:
         return action
 
     def clear_action(self) -> None:
+        """Remove all actions"""
         self._actions.clear()
         dispatcher.send('core_a_list_changed')
 
     @property
     def actions(self) -> List[Action]:
+        """Get actions"""
```

Comment:
Same here, and for `devices`. I think it's better w/o docstrings

Reply:
See Above.

Reply:
Fair, you're right. Maybe you could make it more descriptive? like "Get list of actions"? Same for the devices property